vgen/src/main/resources/io/gapi/vgen/java/java_gapic.yaml

```diff
@@ -1,8 +1,10 @@
 type: io.gapi.vgen.ConfigProto
 language: java
-language_provider: io.gapi.vgen.java.JavaGapicLanguageProvider
-snippet_files:
-- main.snip
-- settings.snip
-fragment_files:
-- fragment.snip
+templates:
+- language_provider: io.gapi.vgen.java.JavaGapicCodeLanguageProvider
+  snippet_files:
+  - main.snip
+  - settings.snip
+- language_provider: io.gapi.vgen.java.JavaGapicFragmentLanguageProvider
+  snippet_files:
+  - fragment.snip
```

Comment:
@pongad reports that this isn't actually used anywhere yet. Vkit snippet generation (snippets within comments) is performed by the main Gapic codegen. I think we should either remove this fragment-gen config or put it in its own file - the process which would make use of this would run independently of GAPIC and its output would be used by different tools.